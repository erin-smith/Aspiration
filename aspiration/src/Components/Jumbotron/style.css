.jumbotron {
    padding-top: 120px;
    height: 300;
    clear: both;
    text-align: center;
    background-color: aquamarine;
  }
  