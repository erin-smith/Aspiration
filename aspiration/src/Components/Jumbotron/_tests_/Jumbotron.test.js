import * as React from "react";
import Jumbotron from "../index";
import renderer from "react-test-renderer";

describe("Jumbotron", () => {
  test("Should match the snapshot", () => {
    const tree = renderer.create(<Jumbotron />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
