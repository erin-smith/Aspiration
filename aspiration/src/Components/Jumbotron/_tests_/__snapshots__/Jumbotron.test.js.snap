// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron Should match the snapshot 1`] = `
<div
  className="jumbotron mt-4"
/>
`;
